xp/sitecore/Dockerfile

```diff
@@ -25,7 +25,7 @@ ARG SOLR_PORT=8983
 ARG SITECORE_PACKAGE
 
 COPY files/*.pfx /Files/
-COPY files/license.xml /Files/
+COPY licenses/license.xml /Files/
```

Comment:
maar 1 license, noem de directory: `license`

Deze COPY mag je pas net voor het RUN command doen waarin je 'm verwijdert volgens mij. Anders zit 'ie al in de image laag ervoor, en kan iemand 'm toch achterhalen.